### PR TITLE
add a small section kube ingress

### PR DESCRIPTION
**Pull request by szuecs**
A tiny section mentioning Kubernetes ingress with some links to an example manifest.

### CODE REVIEW

**Comment by aryszka** [comment]
👍

**Comment by lmineiro** [comment]
👍

File: readme.md [hunk 1]
```diff
@@ -59,6 +59,13 @@ Start Skipper and make an HTTP request:
     skipper -routes-file example.eskip &
     curl localhost:9090/hello
 
+##### Kubernetes Ingress
+
+Skipper can be used to run as Ingress implementation. A [production
```
Comment (hjacobs) [inline readme.md]: sounds wrong (missing "an")

Comment (szuecs) [inline readme.md]: @hjacobs Where?
"as an Ingress" or "An production example"?

Comment (lmineiro) [inline readme.md]: "as an Ingress"

**Pull request merged**